scripts/linux/pkg/linux-packaging.md

```diff
@@ -0,0 +1,88 @@
+# RavenDB installation from package on Linux
+
+## What the package does on install
+
+- scatter the files in the default locations
+
+- install systemd service
+
+- add soft links to binaries
+
+- add `ravendb` user and group
+
+- set file permissions properly
+
+- setcap on the Raven.Server
+
+
+## Supported use cases
+
+### Ran under systemd
+The default is when RavenDB is used as a systemd daemon and controlled with systemd.
+The following directory layout (initialized by the package installation process) and permissions is used in this scenario:
+
+#### Filesystem locations and permissions
+
+- settings - `/etc/ravendb/settings.json`
+
+- data - `/var/lib/ravendb/data`
+
+- security (e.g. certificate)  - `/var/lib/ravendb/security`
+
+- logs  - `/var/log/ravendb/logs`
+
+- audit logs  - `/var/log/ravendb/audit`
+
+- binaries (current Server dir) - `/usr/local/ravendb`
+
+- rvn link - `/usr/bin/rvn -> /usr/local/ravendb/rvn`
+
+- ravendb - `/usr/bin/ravendb -> /usr/local/ravendb/Raven.Server` 
```

Comment:
The ability to run this independently as a stand alone server isn't really attractive, what scenario are we trying to enable here?